File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_ckprusrn.java
// EXPECTED: 20
// Reference emit for CKPRUSRN — PROCEDURE USING + CALL TAKE path.
public final class CkprusrnRef {
  public static void main(String[] args) {
    String entry = "T";
    int phase = switch (entry) {
      case "I" -> 0;
      case "T" -> 20;
      case "C" -> 30;
      case "R" -> 40;
      default -> 99;
    };
    System.out.printf("%02d%n", phase);
  }
}

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_ckprusrn.py
"""Reference emit for CKPRUSRN — CALL/USING TAKE path → phase 20."""
# EXPECTED: 20
ENTRY = "T"
PHASE = {"I": 0, "T": 20, "C": 30, "R": 40}.get(ENTRY, 99)
print(f"{PHASE:02d}")
